sources/api/apiclient/src/lib.rs

```diff
@@ -140,3 +142,12 @@ where
 
     Ok((status, body))
 }
+
+/// Generates a random ID, affectionately known as a 'rando'.
+pub(crate) fn rando() -> String {
+    thread_rng()
+        .sample_iter(&Alphanumeric)
+        .take(16)
+        .map(char::from)
+        .collect()
+}
```

Comment:
Should we split this into its own crate? We've got a copy in a few places:
* models build script
* storewolf
* migrator